tests/test_layers.py

```diff
@@ -78,3 +78,32 @@ def test_get_country_data_level_1(self, mocker, capsys, multipolygon):
 
         assert getattr(adm_area, "geometry", None) is None
         assert getattr(adm_area, "adm_name", None) is None
+
+
+@pytest.fixture
+def adm_area(mocker, multipolygon):
+    mocker.patch("gpbp.layers.AdmArea._get_country_data")
+    adm_area = AdmArea(country="Timor-Leste", level=0)
+    adm_area.geometry = multipolygon
+    adm_area.adm_name = "Timor-Leste"
+    return adm_area
+
+class TestAdmAreaGetFacilities:
+    def test_get_facilities_valid_method(self, mocker, adm_area):
+        mock_facilities_src = mocker.patch("gpbp.layers.FACILITIES_SRC", {"osm": mocker.Mock()})
+        tags = {"building":"hospital"}
```

Comment:
Simply a matter of taste: `{"building":"hospital”}` is used three times. If it becomes, I don’t know, `{“amenity":"hospital”}`, then we need to change it all over. Extract constant?

On the other hand, this way is perhaps more readable. Dealer’s choice

Reply:
Agreed. I don't know what the best practice would be here, a fixture, a static method, or a class attribute. I just went with fixture for no particular reason.